Makefile.work

```diff
@@ -115,6 +115,10 @@ rules/config.user:
 include rules/config
 -include rules/config.user
 
+ifneq ($(DEFAULT_CONTAINER_REGISTRY),)
+override DEFAULT_CONTAINER_REGISTRY := $(DEFAULT_CONTAINER_REGISTRY)/
```

Comment:
"container registry" is a weird name. Do you mean [docker registry](https://docs.docker.com/registry/) ?

Reply:
yes, it seems docker registry is better.
container registry comes from azure container registry.
When I enable ACR in pipeline, it is a chance to change.